lambdas/db-migration/src/migrations/20200813174231_create_collections_table.ts

```diff
@@ -3,19 +3,27 @@ import * as Knex from 'knex';
 export const up = async (knex: Knex): Promise<void> =>
   knex.schema.createTable('collections', (table) => {
     table.increments('cumulusId').primary();
-    table.string('name').notNullable();
-    table.string('version').notNullable();
-    table.string('sampleFileName').notNullable();
-    table.text('granuleIdValidationRegex').notNullable();
-    table.text('granuleIdExtractionRegex').notNullable();
-    table.jsonb('files').notNullable();
-    table.string('process');
-    table.string('url_path');
-    table.enum('duplicateHandling', ['error', 'replace', 'skip', 'version']);
-    table.boolean('reportToEms');
-    table.boolean('ignoreFilesConfigForDiscovery');
-    table.jsonb('meta');
-    table.jsonb('tags');
+    table.string('name').comment('Collection short_name registered with the CMR').notNullable();
+    table.string('version').comment('The version registered with the CMR').notNullable();
+    table.string('sampleFileName').comment('Example filename for this collection').notNullable();
+    table.string('granuleIdValidationRegex').notNullable().comment(
+      'The regular expression used to validate the granule ID extracted from filenames '
+      + 'according to the granuleIdExtraction'
+    );
+    table.string('granuleIdExtractionRegex').comment('The regular expression used to extract the granule ID from filenames').notNullable();
+    table.jsonb('files').comment('List of collection file definitions').notNullable();
+    table.string('process').comment('Name of the docker process to be used, e.g. modis, aster');
+    table.string('url_path').comment('The folder (url) used to save granules on S3 buckets');
+    table.enum('duplicateHandling', ['error', 'replace', 'skip', 'version']).comment(
+      'Duplicate handling behavior for this collection'
+    );
+    table.boolean('reportToEms').comment('Flag to set if this granule should be reported to EMS');
+    table.boolean('ignoreFilesConfigForDiscovery').comment(
+      'When true, ignore the collection files config list for determining which files to ingest for a granule'
+      + 'When false, ingest only files that match a regex in the colletion files config list'
+    );
+    table.jsonb('meta').comment('Collection meta object');
+    table.jsonb('tags').comment('Collection tags object');
```

Comment:
This is usually an array of tags

Reply:
I'm assuming that's in reference to the .comment?    I'm unclear on how descriptive we should be with jsonb - in most databases, the typings are self explanatory for other columns. 

How about: 

31cdde802